parcels/application_kernels/TEOSseawaterdensity.py

```diff
@@ -22,7 +22,7 @@ def PolyTEOS10_bsq(particle, fieldset, time):
     Oceanic Technology, 20, 730-741.
     '''
 
-    Z = - particle.depth  # note: use negative depths!
+    Z = math.fabs(particle.depth)
```

Comment:
```suggestion
    Z = - math.fabs(particle.depth)
```
I think the point was that `Z` should always be a negative number?

Reply:
I was confused about that because of the comment on the same line.